### ISSUE TITLE
Persist 'Show deprecated applications' checkbox checked state 

### ISSUE THREAD

**Issue by Riju19** [mozilla/glean-dictionary#210]
Currently, when a user selects the checkbox to show deprecated applications and clicks on an application, on returning back to the main applications page, the state of this checkbox gets reset to being unselected.

This task is to persist the state of the checkbox, so that on returning back, the checkbox would be pre-populated to the last state, and therefore, have the list of applications (along with deprecated ones) already rendered.

![image](https://user-images.githubusercontent.com/24811075/99884856-2e918e00-2c57-11eb-9842-0d7d41e5a446.png)

**Comment by wlach** [comment]
Yup, definitely. Ideally all application state like this will be persisted in the URL eventually.

**Comment by wlach** [comment]
Hey @Riju19, assuming you don't have time to work on this (no worries!) Note that we won't have any deprecated application groupings in the main view after #426 merges, so this is much lower priority.
